dashboard/app/views/levels/editors/fields/_starter_assets.html.haml

```diff
@@ -5,4 +5,6 @@
   #starter-assets-editor
 
 - content_for :body_scripts do
+  -# Including the lab2 locale bundle since the starter assets modal references a few localized strings.
+  %script{src: webpack_asset_path("js/#{js_locale}/lab2_locale.js")}
```

Comment:
Had to add this to get some translated content to work in levelbuilder. LMK if there's a more general lab2 HAML place to put this script tag, I couldn't find one.